plz.el

```diff
@@ -449,11 +449,14 @@ NOQUERY is passed to `make-process', which see.
                       (progn
                         (write-region (point-min) (point-max) filename)
                         (funcall then filename))
+                    (file-already-exists
+                     (funcall then (make-plz-error :message (format "error while writing to file %S: %S" filename err))))
                     ;; In case of an error writing to the file, delete the temp file
                     ;; and signal the error.  Ignore any errors encountered while
                     ;; deleting the file, which would obscure the original error.
                     (error (ignore-errors
-                             (delete-file filename))
+                             (when (file-exists-p filename)
```

Comment:
Since we're already using `ignore-errors`, is it necessary to check whether the filename exists?

Reply:
Of course, you're right.  I'll rebase and ditch this commit.